contributing.md

```diff
@@ -21,11 +21,11 @@ If you want to add or update an icon, please open a pull request, making sure th
 ### Technical:
 
 1. Icons must be SVG v1.1
-1. Icons must not contain any ClipPaths. This is because the conversion of SVG to PNG in v1 of the Image Service does not work if ClipPaths are present.
+1. Icons must not contain any ClipPaths.
```

Comment:
@rowanmanning do you think we should keep this?

Reply:
Oh no, this isn't a requirement any more. Feel free to remove 👍